frontend/turbo.json

```diff
@@ -22,10 +22,8 @@
     },
     "fmt": {},
     "test": {
+      "dependsOn": ["^build"],
       "outputs": []
-    },
-    "@liam-hq/cli#test": {
-      "dependsOn": ["^build"]
```

Comment:
📝 Applied across the all, as build was needed first in `erd-core` as well as `cli`.